pkgs/swiftgen/swift2objc/lib/src/ast/_core/shared/referred_type.dart

```diff
@@ -2,22 +2,30 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'package:swift2objc/src/ast/_core/interfaces/objc_annotatable.dart';
+
 import '../interfaces/declaration.dart';
 
 /// Describes a type reference in declaration of Swift entities (e.g a method return type).
 /// See `DeclaredType` and `GenericType` for concrete implementation.
-abstract class ReferredType {
-  String id;
+sealed class ReferredType {
+  final String id;
+  abstract final bool isObjcRepresentable;
```

Comment:
Nit: capitalize the C, `isObjCRepresentable`.

Reply:
fixed